7_react/react1/week1/vite-project/ToDoItems.jsx

```diff
@@ -0,0 +1,17 @@
+
+const ToDoItems = [
```

Comment:
It's common convention that variables should always start with a lowercase letter (snake_case, kebab-case or pascalCase is ok). The Capital letters are reserved for React Componentes or Types (foreshadowing upcoming lessons :))